public/app/plugins/datasource/grafana-pyroscope-datasource/dataquery.cue

```diff
@@ -43,7 +43,7 @@ composableKinds: DataQuery: {
 				maxNodes?:           int64
 				#PyroscopeQueryType: "metrics" | "profile" | *"both" @cuetsy(kind="type")
         // If set to true, the response will contain annotations
-        annotations?: bool
+        annotations?: bool | *true
```

Comment:
Could we keep it false by default? In Profiles Drilldown we added explicit places where we want to show annotations (e.g. in the main graph rather on each breakdown) - https://github.com/grafana/profiles-drilldown/pull/522

Reply:
Makes sense 👍  We can keep it on by default for Explore queries. I'll need to double check if it's gonna affect current queries coz we do something funky with normalizeQuery https://github.com/grafana/grafana/blob/bbf01a638345e42c0f48dfcc743efbfbb7fedaeb/public/app/plugins/datasource/grafana-pyroscope-datasource/datasource.ts#L161

With `{ ...defaultQuery, ...query }` it may automatically add annotations to existing dashboard queries 🤔  Just to be safe we can always split `defaultQuery` into 2 objects to make sure normalizeQuery stays intact and we just have a different default for Explore in getDefaultQuery https://github.com/grafana/grafana/blob/bbf01a638345e42c0f48dfcc743efbfbb7fedaeb/public/app/plugins/datasource/grafana-pyroscope-datasource/datasource.ts#L151